packages/gatsby/src/schema/print.js

```diff
@@ -185,7 +185,7 @@ const {
   DEFAULT_DEPRECATION_REASON,
 } = require(`graphql`)
 const { printBlockString } = require(`graphql/language/blockString`)
-const { flatMap } = require(`graphql/polyfills/flatMap`)
+const flatMap = require(`graphql/polyfills/flatMap`).default
```

Comment:
can we actually remove this import and use `Array.flatMap` further down on line 381?

Reply:
I would say yes but it is not my decision to make. Native flatMap is [only available with node 11+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) so I suspect this may break for quite a few people.

Reply:
Let's use https://www.npmjs.com/package/array.prototype.flatmap ourselves instead of trying to import it from graphql package internals or just rewrite it the code that use it a little bit (i.e. use lodash `_.flatten` and map after that)

Reply:
I changed the import to use `array.prototype.flatmap` @pieh. Is it ok?

Reply:
I think so, I'll do test run in a sec and hopefully merge this in